lib/node_modules/@stdlib/math/base/special/cabs/package.json

```
@@ -66,5 +66,141 @@
     "complex",
     "cmplx",
     "number"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "cabs",
+      "alias": "cabs",
+      "pkg_desc": "compute the absolute value of a complex number",
+      "desc": "computes the absolute value of a complex number",
+      "short_desc": "absolute value",
+      "parameters": [
+        {
+          "name": "z",
+          "desc": "input value",
+          "type": {
+            "javascript": "Complex128",
+            "jsdoc": "Complex128",
+            "c": "stdlib_complex128_t",
+            "dtype": "complex128"
+          },
+          "domain": null,
+          "rand": {
+            "prng": "random/base/uniform",
+            "parameters": [
+              [
+                -10,
+                10
+              ],
+              [
+                -10,
+                10
+              ]
+            ]
+          },
+          "example_values": [
+            {
+              "re": -3.14,
+              "im": -1.5
+            },
+            {
+              "re": 0,
+              "im": 0
+            },
+            {
+              "re": -1.5,
+              "im": 2.5
+            },
+            {
+              "re": 2.5,
+              "im": -1.5
+            },
+            {
+              "re": 0,
+              "im": -3.7
+            },
+            {
+              "re": 4.2,
+              "im": 0
+            },
+            {
+              "re": 21.2,
+              "im": 3
+            },
+            {
+              "re": 11,
+              "im": -5
+            },
+            {
+              "re": 33,
+              "im": -14.67
+            },
+            {
+              "re": -42,
+              "im": 9.3
+            },
+            {
+              "re": -3,
+              "im": 3
+            },
+            {
+              "re": 73,
+              "im": 31
+            },
+            {
+              "re": -2.45,
+              "im": 1.23
+            },
+            {
+              "re": 2.45,
+              "im": -1.23
+            },
+            {
+              "re": 1.77,
+              "im": -3.14
+            },
+            {
+              "re": -7.5,
+              "im": 8.2
+            },
+            {
+              "re": 5.5,
+              "im": -12.3
+            },
+            {
+              "re": -15.8,
+              "im": 0.4
+            },
+            {
+              "re": 0.99,
+              "im": -0.99
+            }
+          ]
+        }
+      ],
+      "output_policy": "real_floating_point_and_generic",
+      "returns": {
+        "desc": "absolute value",
+        "type": {
+          "javascript": "number",
+          "jsdoc": "number",
+          "c": "double",
+          "dtype": "float64"
+        }
+      },
+      "keywords": [
+        "abs",
+        "absolute",
+        "magnitude",
+        "modulus",
+        "complex",
```

```suggestion
        "modulus",
        "distance",
        "complex",
```
Ref: https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/cabs/package.json#L65